Tests/MongoSwiftTests/ReadPreferenceTests.swift

```diff
@@ -67,11 +67,11 @@ final class ReadPreferenceTests: MongoSwiftTestCase {
     }
 
     func testEquatable() throws {
-        expect(ReadPreference(.primary)).to(equal(ReadPreference(.primary)))
```

Comment:
similar to the other cases, we're testing the `Equatable` conformance here, not the default `ReadPreference` behavior, so I think this test can be left as-is.